config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-release-v1alpha3.yaml

```diff
@@ -29,10 +29,10 @@ periodics:
             memory: "9Gi"
   annotations:
     testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
-    testgrid-tab-name: capz-periodic-conformance-v1alpha3-release-0-4
+    testgrid-tab-name: capz-periodic-conformance-v1alpha3
     testgrid-alert-email: k8s-infra-staging-cluster-api-azure@kubernetes.io
-    description: Runs conformance & node conformance tests on a stable k8s version with latest cluster-api-provider-azure
-- name: periodic-cluster-api-provider-azure-conformance-v1alpha3-with-ci-artifacts-release-0-4
+    description: Runs conformance & node conformance tests on a stable k8s version with cluster-api-provider-azure release-0.4 (v1alpha3)
+- name: periodic-cluster-api-provider-azure-conformance-v1alpha3-with-ci-artifacts
```

Comment:
can we rename this to `periodic-cluster-api-provider-azure-conformance-with-ci-artifacts-v1alpha3` to be consistent?